cmd/vcfAFS/vcfAFS.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/vertgenlab/gonomics/fileio"
+	"github.com/vertgenlab/gonomics/popgen"
+	"log"
+)
+
+func vcfAFS(vcfFile string, outFile string) {
+	g := popgen.GVCFToAFS(vcfFile)
+	f := popgen.AFSToFrequency(g)
+	out := fileio.EasyCreate(outFile)
+	defer out.Close()
+	for i := 0; i < len(f); i++ {
+		fmt.Fprintf(out, "%f\n", f[i])
+	}
+}
+
+func usage() {
+	fmt.Print(
+		"vcfAFS - Returns allele frequency spectrum information in a text file for graphing.\n" +
+			"vcfAFS - vcfFile.vcf outFile.txt")
```

Comment:
Could you put a newline at the end of this line. When i run the cmd it gets pushed up against the expected args fatal and is a bit hard to read.